app/collins/models/Asset.scala

```diff
@@ -266,7 +266,8 @@ object Asset extends Schema with AnormAdapter[Asset] with AssetKeys {
     val findLocations = find(PageParams(0, 50, "ASC", "tag"), AttributeResolver.emptyResultTuple, instanceFinder)
       .items
       .collect { case a: Asset => a }
-      .filter { _.tag != MultiCollinsConfig.thisInstance }
+      // filter out the local instance so that we don't perform both a local and a remote query to it
+      .filter(asset => { !MultiCollinsConfig.thisInstance.contains(asset.tag) })
```

Comment:
This isn't a substring match but a function on the `Option` type. It does an equality test on the string within the `Option[String]` that is `MultiCollinsConfig.thisInstance` (or returns false if the option is `None`). It is not a substring match so I think this should be fine.

```
scala> Some("dc1").contains("dc1")
res1: Boolean = true

scala> Some("dc1").contains("dc")
res0: Boolean = false

scala> None.contains("dc1")
res2: Boolean = false
```